qa/ccs-unavailable-clusters/build.gradle

```diff
@@ -5,16 +5,13 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-apply plugin: 'elasticsearch.internal-testclusters'
-apply plugin: 'elasticsearch.standalone-rest-test'
-apply plugin: 'elasticsearch.rest-test'
-apply plugin: 'elasticsearch.test-with-dependencies'
+apply plugin: 'elasticsearch.internal-java-rest-test'
 
-testClusters.matching { it.name == "integTest" }.configureEach {
+testClusters.matching { it.name == "javaRestTest" }.configureEach {
```

Comment:
Not sure why we used`matching` here when there's only ever one. Wouldn't `testClusters.named()` be more concise?

Reply:
this is because we create those cluster lazy as part of the task creation. That means that taskClusters.named would not find the cluster yet when hitting this config block initially